src/Makefile

```diff
@@ -822,8 +822,13 @@ endif
 ifeq (,$(filter ${DISABLE_STRIP_SYMBOLS},YES yes y Y 1))
 ifeq (${uname_S}, Linux)
 ifeq (${TARGET}, winagent)
-	@if test -n "${WIN_STRIP_TOOL_PATH}" -a -d "${WIN_STRIP_TOOL_PATH}" -a -f "${WIN_STRIP_TOOL_PATH}/cv2pdb.exe" && command -v wine > /dev/null 2>&1; then \
-		$(foreach binary, ${EXECUTABLE_FILES}, (${MING_BASE}objdump -h $(binary) | grep -q ".debug_info" && wine ${WIN_STRIP_TOOL_PATH}/cv2pdb.exe $(binary) $(binary) ${SYMBOLS_PATH}/$(basename $(notdir $(binary))).pdb) || true;) \
+	@if test -n "${WIN_STRIP_TOOL_PATH}"; then \
+		if test -d "${WIN_STRIP_TOOL_PATH}" -a -f "${WIN_STRIP_TOOL_PATH}/cv2pdb.exe" && command -v wine > /dev/null 2>&1; then \
```

Comment:
I think that the use of `-a` and `-f` in the `if` condition is redundant

> [ -a FILE ]  - True if FILE exists.
> [ -f FILE ]  - True if FILE exists and is a regular file.

Flags `-f` is enough

Reply:
With the `test` command the meaning of the `-a` flag is different. Please @RamiroRD discard this change request.